cluster/addons/dns/kube2sky/Makefile

```diff
@@ -18,8 +18,8 @@
 
 .PHONY: all kube2sky container push clean test
 
-TAG = 1.12
-PREFIX = gcr.io/google_containers
+TAG = 3.03.16-1
```

Comment:
Push it and update the tags so we can merge?  Need to merge with @bprashanth's changes